README.md

```diff
@@ -72,143 +75,29 @@ Directives from this repository are automatically tested with the following brow
-
-We are aiming at providing a set of AngularJS directives based on Bootstrap's markup and CSS. The goal is to provide **native AngularJS directives** without any dependency on jQuery or Bootstrap's JavaScript.
-It is often better to rewrite an existing JavaScript code and create a new, pure AngularJS directive. Most of the time the resulting directive is smaller as compared to the original JavaScript code size and better integrated into the AngularJS ecosystem.
-
-### Customizability
-
-All the directives in this repository should have their markup externalized as templates (loaded via `templateUrl`). In practice it means that you can **customize directive's markup at will**. One could even imagine providing a non-Bootstrap version of the templates!
+> **Internet Explorer 8 support**
 
-### Take what you need and not more
+> Since AngularJS 1.3, the Angular team has stopped supporting Internet Explorer 8. From UI Bootstrap version `0.13.0` on, we will update the AngularJS dependency to version 1.3 and by doing so **we will drop all Internet Explorer 8 support**. (New) issues regarding Internet Explorer 8 will be closed.
 
-Each directive has its own AngularJS module without any dependencies on other modules or third-party JavaScript code. In practice it means that you can **just grab the code for the directives you need** and you are not obliged to drag the whole repository.
-
-### Quality and stability
-
-Directives should work. All the time and in all browsers. This is why all the directives have a comprehensive suite of unit tests. All the automated tests are executed on each checkin in several browsers: Chrome, ChromeCanary, Firefox, Opera, Safari, IE9.
-In fact we are fortunate enough to **benefit from the same testing infrastructure as AngularJS**!
-
-## Support
-
-If you are having problems making some directives work, there are several ways to get help:
+## Need help?
+Need help using UI Bootstrap?
 
 * Live help in the IRC (`#angularjs` channel at the `freenode` network). Use this [webchat](https://webchat.freenode.net/) or your own IRC client.
-* Ask a question in [stackoverflow](http://stackoverflow.com/) under the [angular-ui-bootstrap](http://stackoverflow.com/questions/tagged/angular-ui-bootstrap) tag.
-* Write your question in our [mailing list](https://groups.google.com/forum/#!categories/angular-ui/bootstrap).
-
-Project's issue on GitHub should be used discuss bugs and features.
-
-## Contributing to the project
-
-We are always looking for the quality contributions! Please check the [CONTRIBUTING.md](CONTRIBUTING.md) for the contribution guidelines.
-
-### Development
-#### Prepare your environment
-* Install [Node.js](http://nodejs.org/) and NPM (should come with)
-* Install global dev dependencies: `npm install -g grunt-cli karma`
-* Install local dev dependencies: `npm install` while current directory is bootstrap repo
-
-#### Build
-* Build the whole project: `grunt` - this will run `lint`, `test`, and `concat` targets
```

Comment:
@chrisirhc  You cant easily see it in this view but I link to 'development, roadmap etc' in the wiki from the readme. 
See this page:
https://github.com/angular-ui/bootstrap/wiki/Development